reconcile/openshift_namespaces.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+import reconcile.openshift_resources as openshift_resources
+
+import utils.gql as gql
+
+from utils.openshift_resource import ResourceInventory
+from utils.oc import StatusCodeError
+
+from multiprocessing.dummy import Pool as ThreadPool
+from functools import partial
```

Comment:
I just realized we are not following the correct order in other files...

```
import logging
from multiprocessing.dummy import Pool as ThreadPool
from functools import partial

import reconcile.openshift_resources as openshift_resources
import utils.gql as gql
from utils.openshift_resource import ResourceInventory
from utils.oc import StatusCodeError
```

first the native ones, then python packages, and then our libraries. Also I prefer first import and then from.

Reply:
fixed